src/Sulu/Bundle/RouteBundle/Manager/ConflictResolverInterface.php

```diff
@@ -24,4 +24,9 @@ interface ConflictResolverInterface
      * @return RouteInterface
      */
     public function resolve(RouteInterface $route);
+
+    /**
+     * @return string
+     */
+    public function resolveByAttributes(?string $entityClass, ?string $id, string $locale, string $path);
```

Comment:
not sure if adding this is a good idea 😕  maybe it would be better to create a temporary route in the `RouteController`? that would be no bc break and we would not need to maintain a `resolveByAttributes` method. the problem is that we might not have an id at that point